applications/nrf_desktop/src/modules/Kconfig.dvfs

```diff
@@ -7,13 +7,27 @@
 menuconfig DESKTOP_DVFS
 	bool "DVFS module"
 	depends on SOC_NRF54H20_CPUAPP
+	select CLOCK_CONTROL_NRF2
 	default y
 	help
 	  This option enable DVFS module which switches frequency and voltage according
 	  to application needs.
 
 if DESKTOP_DVFS
 
+config DESKTOP_DVFS_FREQ_HIGH
+	int "High frequency"
+	default 320000000
+
+config DESKTOP_DVFS_FREQ_MED
+	int "Medium frequency"
+	default 128000000
+
+config DESKTOP_DVFS_FREQ_LOW
+	int "Low frequency"
+	default 64000000
+
+
 config DESKTOP_DVFS_RETRY_BUSY_TIMEOUT_MS
```

Comment:
Discussed offline. Please check if we still need to retry in application scope (how clock control driver handles cases where DVFS state is misaligned with requests' state). Maybe we could suggest an improvement if the automatic retry is not done in scope of the clock control driver?

Reply:
Yes we still need to handle retry in application scope.